resources/assets/js/store/asyncAction.ts

```diff
@@ -8,6 +8,7 @@ import {
   getJSON,
 } from "redux-api-middleware"; // RSAA = '@@redux-api-middleware/RSAA'
 import { ErrorAction } from "./createAction";
+import moment = require("moment");
```

Comment:
😢

Reply:
Quality sprint, right? ...Right?

Reply:
But I fixed the syntax here at least. `import moment from "moment";`